db/migrate/20220609155556_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.belongs_to :users, null: false, foreign_key: true
```

Comment:
Please create `author_id` as a foreign key instead of `user_id` as it can be seen here in [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) ERD.